<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankDlgClass</class>
 <widget class="QDialog" name="BankDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑银行</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonAdd">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_add.png</normaloff>:/Images/edit_add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonDel">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_remove.png</normaloff>:/Images/edit_remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonEdit">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/Notepad2.png</normaloff>:/Images/Notepad2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonSave">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/Floppy.png</normaloff>:/Images/Floppy.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="buttonCancel">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/editdelete.png</normaloff>:/Images/editdelete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>BankDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>BankDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
